docs/gettingStarted/quickStart.rst

```diff
@@ -113,6 +112,52 @@ For information on using CWL with Toil see the section :ref:`cwl`
 
 .. _CWL User Guide: http://www.commonwl.org/v1.0/UserGuide.html
 
+Running a basic WDL workflow
+----------------------------
+
+The `Workflow Description Language`_ (WDL) is another emerging language for writing workflows that are portable across multiple workflow engines and platforms.
+Running WDL workflows using Toil is still in alpha, and currently experimental.  Toil currently supports basic workflow syntax (see :ref:`wdlSupport` for more details and examples).  Here we go over running a basic WDL helloworld workflow.
+
+#. First ensure that Toil is installed with the
+   ``wdl`` extra (see :ref:`extras`).  ::
+
+        (venv) $ pip install 'toil[wdl]'
+
+   This installs the ``toil-wdl-runner`` executable.
+
+#. Copy and paste the following code block into ``wdl-helloworld.wdl``:
+
+::
+
+        workflow write_simple_file {
+          call write_file
+        }
+        task write_file {
+          String message
+          command { echo ${message} > wdl-helloworld-output.txt }
+          output { File test = "wdl-helloworld-output.txt" }
+        }
+
+   and this code into ``wdl-helloworld.json``::
+
+        {
+          "write_simple_file.write_file.message": "Hello world!"
+        }
+
+#. To run the workflow simply enter ::
+
+        (venv) $ toil-wdl-runner wdl-helloworld.wdl wdl-helloworld.json
+
+   Your output will be in ``wdl-helloworld-output.txt`` ::
+
+        (venv) $ cat output.txt
```

Comment:
the output file name here should be wdl-helloworld-output.txt as specified above.